clang/test/SemaSYCL/scheduler_target_fmax_mhz.cpp

```diff
@@ -0,0 +1,52 @@
+// RUN: %clang_cc1 %s -fsyntax-only -fsycl -fsycl-is-device -triple spir64 -Wno-sycl-2017-compat -verify
```

Comment:
Can't this be done with a single run line without the -DTRIGGER_ERROR

`// RUN: %clang_cc1 %s -fsyntax-only -fsycl -fsycl-is-device -triple spir64 -Wno-sycl-2017-compat -verify -ast-dump | FileCheck %s`

Reply:
Done. Thanks!